tests/e2escenarios/e2e_beta_test.go

```diff
@@ -75,19 +75,14 @@ var _ = Describe("odo core beta flow", func() {
 		helper.CmdShouldPass(odo, append([]string{"push"}, extraArgs...)...)
 
 		dcSession := oc.GetComponentDC("mycomponent", "myapp", project)
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/instance: mycomponent"))
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/component-source-type: local"))
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/name: java"))
-		Expect(dcSession).Should(ContainSubstring("app.kubernetes.io/part-of: myapp"))
-		Expect(dcSession).Should(ContainSubstring("name: mycomponent-myapp"))
+		helper.MatchAllInOutput(dcSession, []string{"app.kubernetes.io/instance: mycomponent", "app.kubernetes.io/component-source-type: local", "app.kubernetes.io/name: java", "app.kubernetes.io/part-of: myapp", "name: mycomponent-myapp"})
```

Comment:
Actually I was talking about this line @prietyc123

Reply:
Ahh... got it.

Reply:
Done